packages/@aws-cdk/core/lib/asset-staging.ts

```diff
@@ -89,10 +89,15 @@ export class AssetStaging extends Construct {
     this.fingerprintOptions = props;
 
     if (props.bundling) {
-      this.bundleDir = this.bundle(props.bundling);
-    }
+      const assetHashIfKnownInAdvance = props.assetHashType !== AssetHashType.BUNDLE
```

Comment:
please add some comments explaining what this is about

Reply:
Done